netsnmp/__init__.py

```diff
@@ -112,6 +111,7 @@ def get(self, oids):
         """
         # Define list to be populated by C API get()
         responses = []
+        self._next = False
```

Comment:
This is already explicitly set false in the SNMPSession \__init__

No need for redundancy here,